### DISCUSSION TITLE
Still Patched :(

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#86] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hi, I saw a new commit for the exploit js.
It made it much easier to access the gui, since my school blocked webstorex. (Idk how they even know)
I can still open the gui and enable/disable some extensions, but all the admin ones (Securly, Dyknow, etc) do not disable.
Do you know if you can fix this, or not?

**Comment by phillipbruh** [comment, marked as answer]
Try going to webstoresjsnsbdbdbdjfkdmendjdkcnfje Or if you have at least 2 or more accounts on chrome you can goto chrome.google.com/webstorex?authuser=1

**Reply by phillipbruh** [reply]
Mark as answer

**Reply by ghost** [reply]
ik about that its just the gui doesn't unblock the admin extenions

**Reply by phillipbruh** [reply]
try using the one thats not gui in bookmark.js

**Reply by ghost** [reply]
yea I tried them all :(

**Comment by synapt0** [comment]
You probably updated your chromebook. The new update patches it.
